forge16/src/main/java/com/envyful/poke/tracker/forge/tracker/PokeTrackerFactory.java

```diff
@@ -45,7 +45,13 @@ public static void addTrackedEntities(PixelmonEntity pixelmon) {
 
             List<EntityData> entities = TRACKED_ENTITIES.computeIfAbsent(entry.getValue().getName().toLowerCase(), ___ -> Lists.newArrayList());
 
-            entities.add(0, EntityData.of(pixelmon));
+
+            EntityData e = EntityData.of(pixelmon);
+
+            // Check if an Entity with the same UUID is already being tracked using lambda
```

Comment:
Don't add inline comments. If the code isn't clear then it needs to be refactored into something that makes it clearer